sdk/openai/Azure.AI.OpenAI/Directory.Build.props

```diff
@@ -19,4 +19,8 @@
     Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
   -->
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  
```

Comment:
TODO: File issue

Reply:
Seems like it would be safer to do a `contains` for "Test" or similar.    What's the purpose of this one?

Reply:
My idea was to only apply this to the main csproj and not the test csproj, but probably not necessary. I can remove.

Reply:
It would be good to have @ralph-msft look at this

Reply:
This needs to stay as-is to avoid overwriting the RequiredTargetFrameworks for the test project in Directory.Build.Common.Props - https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L142

The IsTestProject check is the same as in Directory.Build.Common.Props